libs/dotnet-sdk/Models/FeatureboardOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Featureboard.DotnetSdk.Models;
+
+public class FeatureboardOptions
+{
+  [Required]
+  public required Uri HttpEndpoint { get; set; }
```

Comment:
This will be defaulted, we don't want to have end users having to specify, but it's an option because if someone wants a self hosted instance then we can support that.

Reply:
Resolved